src/kaocha/plugin/alpha/spec_test_check.clj

```diff
@@ -46,24 +86,10 @@
           [nil "--stc-max-size SIZE"        "spec.test.check: Maximum length of generated collections"
            :parse-fn #(Integer/parseInt %)]))
   (config [{:kaocha/keys [tests cli-args] :as config}]
-    (if (and (seq cli-args)
-             (not (some #{:generative-fdef-checks} cli-args)))
+    (if (and (seq cli-args) (not (some #{:generative-fdef-checks} cli-args)))
```

Comment:
Something I noticed here: this should probably use the `:kaocha.testable/id`s specified by all specified `generative-fdef-check` suites instead of this hard-coded ID.